scripts/caretaking/cherrypick-patch.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+/**
+ * This script displays the commands that should be run to cherry-pick merged changes
+ * into the current patch branch.
+ *
+ * Run this command like this: `node scripts/caretaking/cherry-pick-patch.js 2019-07-29` where the
+ * date input is the oldest merged pull request that should be patched.
+ *
+ * An authorization token may optionally be provided to increase the API rate limit. A personal
+ * access token string should be added to the command, e.g.:
+ * node scripts/caretaking/cherrypick-patch.js 2019-07-15 4c32652da9e57b2b2IMATOKENLOL8983B3BK
+ */
+
+/** Required argument: Date of the oldest PR to load, should be in the form of YYYY-MM-DD. */
+const date = process.argv[2];
```

Comment:
Could you just save the last cp'd sha in some file we check in to source control as the default? That way the caretaker could run this as part of doing the release and just check in the updated sha along with the changes to the CHANGELOG.md and package.json